### DISCUSSION TITLE
[Suggestion] Laser Burst Barrel/Addon

### DISCUSSION THREAD

**Discussion by BigBadKangaroo** [tsunamayo/Starship-EVO#4010] [Suggestions]
Hi there,

I don't know if this is ever been mentioned but how about a barrel addon thingy that gives the laser a burst mode?

I.e. a barrel that is 4m long will give it a burst of 4 shots. 
A barrel that is 2m long will give it a burst of 2 shots and so on ... 
The precission would be the same as the laser without any barrel on (maybe a bit better but not much), but of course with the downside that a 4m long barrel will give it a 4sec cooldown.

It's nothing big like the voxel / grid topic, but would be a nice thing to have imo.

**Comment by tsunamayo** [comment]
Make sense, I will think about it. Thx
